vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/server/certs/serving.go

```diff
@@ -26,17 +27,25 @@ type CertKeyFileReference struct {
 	Key  string
 }
 
+type CABundleFileReferences struct {
+	CABundles []string
+}
+
 // DynamicLoader dynamically loads certificates and provides a golang tls compatible dynamic GetCertificate func.
-type DynamicLoader struct {
+type DynamicServingLoader struct {
+	BaseTLSConfig tls.Config
```

Comment:
this one could have a go doc. What is it? What is used from it? What is overridden?